### ISSUE TITLE
Impossible to send a message from a user

### ISSUE THREAD

**Issue by floernoult** [intercom/intercom-java#40]
Hi,

I'm trying to send a message from a user using the Conversation resource. 
So what I do is basically:

```javascript
// Fetch a user from Intercom
User intercomUser = User.find('USER_ID')

// Create a UserMessage and set the body and the user sending it
UserMessage userMessage = new UserMessage().setBody('This is a test!').setUser(intercomUser)

// Create the conversation
Conversation.create(userMessage)
```

This seems to be exactly what is described in the doc, but whatever I try, I keep getting this error
>io.intercom.api.NotFoundException: Resource Not Found

The user is correctly returned, but the error is sent when trying the create the conversation at the last line.

Could you please try on your side and confirm that this code is working for you? I can't figure out the reason why this error is returned.

Thanks a lot.
Florian

**Comment by floernoult** [comment]
According to the Intercom documentation, the JSON message sent with the POST request should be 

```javascript
{
  "from": {
    "type": "user",
    "id": "536e564f316c83104c000020"
  },
  "body": "Hey"
}
```
But while debugging, I find the following JSON body sent in the request:
```javascript
{
  "type": "user_message",
  "message_type": "inapp",
  "created_at": 0,
  "body": "Hey"
  "from": {
    "id": "536e564f316c83104c000020"
  }
}
```
According to Intercom, I think that message_type shouldn't be passed here, and that the user type is missing. Maybe this is an explanation?

**Comment by dehora** [comment]
@floernoult yep, the server wants the see the type field and does not want to see the message type. https://github.com/intercom/intercom-java/issues/41 is a patch for both

**Comment by dehora** [comment]
@floernoult this should be fixed in 1.0.6, thanks for reporting this.

**Comment by floernoult** [comment]
Thanks a lot for the fix, it's working great!
Regards
